charts/descheduler/templates/cronjob.yaml

```diff
@@ -100,6 +103,9 @@ spec:
               volumeMounts:
                 - mountPath: /policy-dir
                   name: policy-volume
+                {{- if .Values.extraVolumeMounts }}
```

Comment:
probably should `extraVolumeMounts` and `extraVolumes` commented out to the value file as well.